### ISSUE TITLE
The Pascal Visual Object Classes Challenge: A Retrospective

### ISSUE THREAD

**Issue by NoahHalonen** [NoahHalonen/NoahHalonen-Capstone_Resources#16]
### Title
The Pascal Visual Object Classes Challenge: A Retrospective

### URL
https://link.springer.com/article/10.1007/s11263-014-0733-5

### Summary 
This article summarizes the PASCAL Visual Object Challenge. It goes into the goals of the competition, along with the makeup of the dataset. It also shows key points and features about the dataset.

### Key Points 
- Three goals; does the image contain an object, where are the objects, and what action is being performed by the objects.
- 11540 images with 31561 objects in training set.
- 2913 images and 6934 objects in validation set.

### Citation
@article{everingham2015pascal,
  title={The Pascal Visual Object Classes Challenge: A Retrospective},
  author={Everingham, Mark and Eslami, S.M. Ali and Van Gool, Luc and others},
  journal={International Journal of Computer Vision},
  volume={111},
  number={1},
  pages={98--136},
  year={2015},
  publisher={Springer},
  doi={10.1007/s11263-014-0733-5},
  url={https://doi.org/10.1007/s11263-014-0733-5},
  note={Received September 12, 2013; Accepted May 23, 2014; Published June 25, 2014; Issue Date: January 2015}
}
